Tests/Command/DropDatabaseDoctrineTest.php

```diff
@@ -42,6 +43,72 @@ public function testExecute() : void
         );
     }
 
+    public function testExecuteWithAlias()
+    {
+        $connectionName = 'default';
+        $dbName         = 'test';
+        $params         = [
+            'url' => 'sqlite:///' . sys_get_temp_dir() . '/test.db',
+            'path' => sys_get_temp_dir() . '/' . $dbName,
+            'driver' => 'pdo_sqlite',
+        ];
+
+        $container = $this->getMockContainer($connectionName, $params);
+
+        $application = new Application();
+        $application->add(new DropDatabaseDoctrineCommand($container->get('doctrine')));
+
+        $command = $application->find('doctrine:database:drop');
+
+        $commandTester = new CommandTester($command);
+        $commandTester->execute(
+            array_merge(['command' => $command->getName(), '-f' => true])
+        );
+
+        $this->assertContains(
+            sprintf(
+                'Dropped database %s for connection named %s',
+                sys_get_temp_dir() . '/' . $dbName,
+                $connectionName
+            ),
+            $commandTester->getDisplay()
+        );
+    }
+
+    public function testExecuteWithExistAliasButNotSupported()
```

Comment:
```suggestion
    public function testExecuteWithExistAliasButNotSupported(): void
```

Also, not sure what you want to say with that method name or what you want to test… could you please explain?

Reply:
Yes, actually, the sqlite driver does not support the `-e` option, the idea is to check if the `DbalException` is thrown when using an "invalid" driver.

Reply:
Ok so `testItThrowsWhenUsingIfExistsWithAnIncompatibleDriver`

Reply:
Improved, thanks 🙂